docs/content/gds/admin/configuration/trtl.en.md

```diff
@@ -1,7 +1,98 @@
+| TRTL_REPLICA_REGION          | string   |         | The region that the replica is assigned to for provenance and geographic compliance.                                  |
+| TRTL_REPLICA_NAME            | string   |         | A unique name that identifies the replica name across peers.                                                          |
+| TRTL_REPLICA_GOSSIP_INTERVAL | duration | 1m      | The mean interval between gossip (synchronization) sessions between trtl peers.                                       |
+| TRTL_REPLICA_GOSSIP_SIGMA    | duration | 5s      | The standard deviation of the jittered interval between gossip sessions.                                              |
+
+If `TRTL_REPLICA_ENABLED` is `true` then `TRTL_REPLICA_PID`, `TRTL_REPLICA_REGION`, and `TRTL_REPLICA_NAME` are required to identify the unique replica in the system.
+
+Replication occurs with bilateral anti-entropy, meaning that after a jittered interval, each replica randomly selects a peer to synchronize with. This is also referred to as a gossip protocol. The `TRTL_REPLICA_GOSSIP_INTERVAL` and `TRTL_REPLICA_GOSSIP_SIGMA` describe a normal distribution of randomly selected synchronization intervals, providing jitter so that the network is not bursty.
+
+### Replica Identification Strategy
+
+When deployed in on a kubernetes cluster, a trtl process running in a container in a pod must identify itself to determine what replica it is so that it can configure itself correctly (particularly if the replica is part of a stateful set). The replica strategy configuration defines how a trtl process bootstraps itself as follows:
+
+| EnvVar                             | Type   | Default | Description                                                                                                              |
+|------------------------------------|--------|---------|--------------------------------------------------------------------------------------------------------------------------|
+| TRTL_REPLICA_STRATEGY_HOSTNAME_PID | bool   | false   | Set to true to use the hostname PID strategy (will be the first strategy tried).                                         |
+| TRTL_REPLICA_HOSTNAME              | string |         | Set the hostname of the process from the environment for the hostname-pid strategy.                                      |
+| TRTL_REPLICA_STRATEGY_FILE_PID     | string |         | Set to the path of a PID file, if not empty uses the file PID strategy (second strategy after hostname-pid).              |
+| TRTL_REPLICA_STRATEGY_JSON_CONFIG  | string |         | Set to the path of a JSON configuration file, if not empty uses the JSON config strategy (third strategy after file-pid). |
+
+The strategies allow the process to identify its PID - either by processing a hostname (e.g. trtl-10) or by reading the pid from a file. The remaineder of the replica can then be configured from the JSON file rather than directly from the environment.
```

Comment:
```suggestion
The strategies allow the process to identify its PID - either by processing a hostname (e.g. trtl-10) or by reading the pid from a file. The remainder of the replica can then be configured from the JSON file rather than directly from the environment.
```